index.html

```diff
@@ -148,44 +154,86 @@ <h2 class="salutation">Tonic</h2>
 
  <!-- This section gives my background -->
 <section class="about-me">
-    <p class="salutation">About Myself </p>
-    <p class="developer">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-
-    <p class="connect">LET'S CONNECT</p>
-
-    <section class="social">
-        <ul >
-            <li><a href="https://twitter.com/home"><img src="images/twitter.jpg" alt="twitter"></a></li>
-            <li> <a href="https://www.linkedin.com/feed/"><img src="images/Linkedin-icon.jpg" alt="Linkedin"></a></li>
-            <li><a href="https://twitter.com/home"><img src="images/Vector.jpg" alt="twitter"></a></li>
-            <li><a href="https://github.com/ausiemass/portfolio"><img src="images/github.jpg" alt="GITHUB"></a></li>
-            <li><a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a></li>
-        </ul> 
-    </section>  
-    <div>
-        <button class="get-resume" type="button">Get my Resume</button>
+    <div class="me">
+        <p class="myself">About Myself </p>
+        <p class="developer">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+
+        <p class="connect">LET'S CONNECT</p>
+
+        <section class="social">
+            <ul >
+                <li><a href="https://twitter.com/home"><img src="images/twitter.jpg" alt="twitter"></a></li>
+                <li> <a href="https://www.linkedin.com/feed/"><img src="images/Linkedin-icon.jpg" alt="Linkedin"></a></li>
+                <li><a href="https://twitter.com/home"><img src="images/Vector.jpg" alt="twitter"></a></li>
+                <li><a href="https://github.com/ausiemass/portfolio"><img src="images/github.jpg" alt="GITHUB"></a></li>
+                <li><a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a></li>
+            </ul> 
+        </section>   
+        
+        <button class="get-resume" type="button">Get my Resume</button>     
```

Comment:
- [ ] As requested by the previous reviewer please decrease the size of the buttons `Get my Resume`. You can fix it by removing `width:50%` it will be ok.